./lib/.libs/libsyslog-ng.so: undefined reference to `sd_notify'

Compiling with systemd support fails on openSUSE 13.1, which has systemd version 208:
```
[   73s]   CCLD   modules/dbparser/libsyslog-ng-patterndb.la
[   73s]   CCLD   lib/tests/test_host_resolve
[   73s]   GEN    syslog-ng/syslog-ng
[   73s] ./lib/.libs/libsyslog-ng.so: undefined reference to `sd_notify'
[   73s] collect2: error: ld returned 1 exit status
[   73s] make[2]: *** [lib/tests/test_host_resolve] Error 1
[   73s] make[2]: *** Waiting for unfinished jobs....
[   73s] /home/abuild/rpmbuild/BUILD/syslog-ng-3.6.3/lib/.libs/libsyslog-ng.so: undefined reference to `sd_notify'
[   73s] collect2: error: ld returned 1 exit status
[   73s] make[2]: *** [syslog-ng/syslog-ng] Error 1
[   73s] make[1]: *** [all-recursive] Error 1
[   73s] make: *** [all] Error 2
```
It compiles fine on openSUSE 13.2, which has sytemd version 210.

--- a/configure.ac
+++ b/configure.ac
@@ -1188,7 +1188,7 @@ if test "x$enable_systemd" = "xyes"; then
                 AC_MSG_ERROR[Detecting system releted systemd-journal library failed])
             fi
             libsystemd_CFLAGS="${LIBSYSTEMD_JOURNAL_CFLAGS} ${libsystemd_daemon_CFLAGS}"
-            libsystemd_LDFLAGS="${LIBSYSTEMD_JOURNAL_LIBS} ${libsystemd_daemon_LIBS}"
+            libsystemd_LIBS="${LIBSYSTEMD_JOURNAL_LIBS} ${libsystemd_daemon_LIBS}"
             AC_SUBST(libsystemd_CFLAGS)
             AC_SUBST(libsystemd_LIBS)
         else

